### ISSUE TITLE
Scroll bars are too short

### ISSUE THREAD

**Issue by JWWeatherman** [JWWeatherman/math_bot#792]
I never noticed this because I use two finger touch to scroll left and right on a touch pad, but if you are trying to grab these bars with an old school mouse and pointer it is really hard.

Even though it will make it uglier we should increase the height by 300% if possible.

<img width="392" alt="screen shot 2019-01-26 at 11 49 48 am" src="https://user-images.githubusercontent.com/32912678/51790265-b217ec00-2160-11e9-9ced-402a6f69af7c.png">

**Comment by giftedprimate** [comment]
This is sized with view port sizing (vmin) so 50% bigger is about right...

![image](https://user-images.githubusercontent.com/33336425/51792991-b3a8da80-2187-11e9-8787-6761e67d9436.png)

**Comment by JWWeatherman** [comment]
Perfect. Only visible when needed?

**Comment by giftedprimate** [comment]
Yes, only when the div is overflowed. Main is visible when 50% of it is filled, this is because the last dropped is always centered if possible.

**Comment by JWWeatherman** [comment]
Awesome.
